File: CommunicationModel/src/RestModels/SensorRegistryRecord.cs

namespace CommunicationModel
{
	public class SensorRegistryRecord
	{
		public string Name { get; set; }
		public string Address { get; set; }
		public int Port { get; set; }
		public long AvailableRecords { get; set; }

		public SensorRegistryRecord()
		{
		}

		public SensorRegistryRecord(string name,
			string address,
			int port,
			int availableRecords = 0)
		{
			this.Name = name;
			this.Address = address;
			this.Port = port;
			this.AvailableRecords = availableRecords;
		}
	}
}

File: DashboardClient/TBoardRabbitAdapter/src/main/java/broker/messages/incoming/SensorRegistryRecord.java
package broker.messages.incoming;

public class SensorRegistryRecord {

	public String Name;
	public String Address;
	public int Port;
	public long AvailableRecords;

	public SensorRegistryRecord(
			String name,
			String address,
			int port,
			int availableRecords) {

		this.Name = name;
		this.Address = address;
		this.Port = port;
		this.AvailableRecords = availableRecords;
	}
}
